.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,5 +11,5 @@ Please list the specific changes involved in this pull request.
 - [ ] If necessary, more likely in a feature request than a bug fix
   - [ ] Unit Tests updated or fixed
   - [ ] Docs/guides updated
-  - [ ] Example created ([starter template on JSBin](http://jsbin.com/liwecukasi/edit?html,output))
+  - [ ] Example created ([starter template on JSBin](http://jsbin.com/vokipos/8/edit?html,output))
```

Comment:
I think this should be http://jsbin.com/nofafidabo/edit?html,output

Reply:
Yeah vim copy mistake